tests/snippets/exit.py

```diff
@@ -0,0 +1,4 @@
+from testutils import assert_raises
```

Comment:
by convention, `tests/snippets/builtin_exit.py` looks better name of this test file.

Reply:
I think this file(exit.py) can hold sys.exit() test. Would be better if separating them?